rns_shell/platform/linux/TaskLoop.h

```diff
@@ -27,6 +27,7 @@ class TaskLoop {
     void waitUntilRunning();
 
     void dispatch(Func fun);
+    void scheduleDispatch(Func fun, long long timeout);
```

Comment:
can we have a comment to state the time unit expected

Reply:
Done